mobile/lib/widgets/custom_widgets.dart

```diff
@@ -552,18 +552,20 @@ class CustomShowcaseWidget extends StatelessWidget {
     required this.description,
     required this.child,
     this.customize,
-    this.descriptionWidth = 200,
-    this.descriptionHeight = 20,
+    this.descriptionWidth,
+    this.descriptionHeight,
   });
 
   final GlobalKey showcaseKey;
   final Widget child;
   final ShowcaseOptions? customize;
   final String description;
-  final double descriptionWidth, descriptionHeight;
+  final double? descriptionWidth, descriptionHeight;
```

Comment:
Thanks @BenjaminSsempala 
Just curious, what is the motivation for this?

Is this some way of accepting double or any other type? If yes, why? 


Thanks again.

Reply:
Thanks @BenjaminSsempala